oshi-core/src/main/java/oshi/software/os/mac/MacFileSystem.java

```diff
@@ -107,6 +106,9 @@
                     if (volume.equals("devfs") || volume.startsWith("map ")) {
                         continue;
                     }
+                    if (localOnly && (volume.startsWith("localhost:") || volume.startsWith("//"))) {
```

Comment:
I'll do some testing later but I'm iffy about this filter.  We do have a `LOCAL_DISK` pattern matcher for the `/dev` path that might be appropriate here.

Reply:
Yep, this is incomplete.  There's at least `smb://192.168.foo.bar/disk` for example and a `foobar@oxfuse`.   And the `/dev` search might not screen out some non-local systems.

Some options are to parse `df -l` similar to the *nixes, parse `mount` looking for the 'local' flag, or use the `f_fstypename` field to grab the string for the same test as the other *nix systems.

I'll dig into the source code for `mount` or `df` later to see if I can figure out what it uses internally.

Reply:
Aha, we have an `f_flags` field already, which we can bitmask with `MNT_LOCAL` (`0x00001000`).  Which makes me realize that perhaps we should reopen #273.

Reply:
Here's the actual code implementing the `l` flag in `df -l`

```
if (lflag && (mntbuf[i].f_flags & MNT_LOCAL) == 0)
			continue;
```